### ISSUE TITLE
/dev/log don't exist in some distros.

### ISSUE THREAD

**Issue by gilgamezh** [PyAr/fades#28]
I just tried the branch `xdg_optional` and I got this error:

```
humitos@manuelkaufmann:~$ ./fades/bin/fades --verbose src/check_python_tutorial.py 
Traceback (most recent call last):
  File "/usr/lib/python3.4/logging/handlers.py", line 820, in _connect_unixsocket
    self.socket.connect(address)
FileNotFoundError: [Errno 2] No such file or directory

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./fades/bin/fades", line 45, in <module>
    main.go(version, sys.argv)
  File "/home/humitos/fades/fades/main.py", line 97, in go
    l = logger.set_up(log_level)
  File "/home/humitos/fades/fades/logger.py", line 28, in set_up
    handler = logging.handlers.SysLogHandler(address='/dev/log')
  File "/usr/lib/python3.4/logging/handlers.py", line 803, in __init__
    self._connect_unixsocket(address)
  File "/usr/lib/python3.4/logging/handlers.py", line 831, in _connect_unixsocket
    self.socket.connect(address)
FileNotFoundError: [Errno 2] No such file or directory
humitos@manuelkaufmann:~$
```

Original reporte by @humitos on #20

**Comment by gilgamezh** [comment]
related: http://serverfault.com/questions/545622/no-more-logging-after-upgrade-to-debian-wheezy

**Comment by gilgamezh** [comment]
@facundobatista I left this issue open, because I did'n make a new branch. Is a very little change. 
Close it if you think that it is ok.

**Comment by gilgamezh** [comment]
Solved on 11b7de3076920d6b6a0beb03dec91d59e60f3145
